packages/scenario/test/secondary-instances.test.ts

```diff
@@ -53,5 +53,38 @@ describe('Secondary instances', () => {
 				});
 			});
 		});
+
+		describe('equivalent predicate expressions in repeats', () => {
+			/**
+			 * **PORTING NOTES**
+			 *
+			 * 1. Rephrase?
```

Comment:
I added a comment referencing the original test name's suffix `doNotGetConfused`, and rephrased this test's description with one I think makes sense for the intent (and BDD-ish format). In the commit note for this, I added that others like this can be updated as we work on them. I think that's probably the most efficient use of time for these, and I'll just 👍 to acknowledge others on the same `doNotGetConfused` theme.